doc/changelog/02-specification-language/18014-master+binder_constr-level10.rst

```diff
@@ -0,0 +1,6 @@
+- **Changed:**
+  :token:`term_forall_or_fun`, :token:`term_let`, :token:`term_fix`,
+  :token:`term_cofix` and :token:`term_if` from :token:`term` at level 200
+  to :token:`term10` at level 10
```

Comment:
The changelog could probably explain what are the reasons for this change and/or what will be its impact for the users.

Reply:
What about: "This is a first step towards getting rid of the recovery mechanism of camlp5/coqpp. The impact will mostly be limited to additional parentheses in rare cases."